WireAPI/Tests/WireAPITests/Helpers/RequestSnapshotHelper.swift

```diff
@@ -78,12 +81,38 @@ struct RequestSnapshotHelper<Builder: APIBuilder> {
         let builder = Builder(httpClient: httpClient)
         let sut = builder.makeAPI(for: apiVersion)
         try? await block(sut)
+
         let request = try XCTUnwrap(httpClient.receivedRequest, "no request was generated")
+        let name = "v\(apiVersion.rawValue)"
+
+        try verifyRequest(request: request,
+                          resourceName: name,
+                          file: file,
+                          function: function,
+                          line: line)
+
+    }
+
+    /// Snapshot test a given request
+    /// - Parameters:
+    ///   - request: httpRequest to verify
+    ///   - resourceName: name of the file containing the expected request description
+    ///   - file: The file invoking the test.
+    ///   - function: The method invoking the test.
+    ///   - line: The line invoking the test.
 
+    @MainActor
+    func verifyRequest(
```

Comment:
With this new method running the snapshot library the other one can be executed without the annotation and be private:
```swift
    private func verifyRequest(
        apiVersion: APIVersion,
        when block: (Builder.API) async throws -> Void,
        file: StaticString = #file,
        function: String = #function,
        line: UInt = #line
    )
```
Then only a new `await` is required by the compiler to call the `@MainActor`.